sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormRecognizerAudience.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Collection;
+
+/** Defines values for FormRecognizerAudience. */
+public class FormRecognizerAudience extends ExpandableStringEnum<FormRecognizerAudience> {
+    /** Static value AZURE_RESOURCE_MANAGER_CHINA for FormRecognizerAudience. */
+    public static final FormRecognizerAudience AZURE_RESOURCE_MANAGER_CHINA = fromString("https://cognitiveservices.azure.cn/.default");
+
+    /** Static value AZURE_RESOURCE_MANAGER_GOVERNMENT for FormRecognizerAudience. */
+    public static final FormRecognizerAudience AZURE_RESOURCE_MANAGER_GOVERNMENT = fromString("https://cognitiveservices.azure.us/.default");
```

Comment:
I'm a little confused about why these are named `AZURE_RESOURCE_MANAGER_*` and not something that relates them to cognitive services given their values.

Reply:
The usage of this would look something like `FormRecognizerAudience.AZURE_RESOURCE_MANAGER_GOVERNMENT` 
And hence I think making the enum vars naming Formrecognizer specific would make it repetitive?